polkadot/runtime/parachains/src/hrmp.rs

```diff
@@ -270,6 +274,10 @@ pub mod pallet {
 		/// implementation should be the same as `Balance` as used in the `Configuration`.
 		type Currency: ReservableCurrency<Self::AccountId>;
 
+		/// The default channel size and capacity to use when opening a channel to a system
+		/// parachain.
+		type DefaultChannelSizeAndCapacityWithSystem: Get<(u32, u32)>;
```

Comment:
Can you please mention the exact values that this holds? So `proposed_max_message_size` and `proposed_max_capacity`.

Reply:
you mean the value that will be used in the relaychain runtime? I don’t exactly have a number in mind but if I have to provide one, I will just make it half of the max value

Reply:
Yea that is fine as a comment as well i guess. like "good values for these would be half of the maximum".

Reply:
we haven’t had discussions about what the value will be and I don’t want to imply anything here. I am deferring the decision until the PR to the relaychain runtime to upgrade polkadot-sdk